spec/features/images_feature_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'rails_helper'
+
+feature 'Images' do
+
+  before do
+    sign_up_and_sign_in
+  end
+
+  context 'Website on initialization' do
+
+    scenario 'should display a prompt to add an image' do
+      visit '/images'
+      expect(page).to have_content 'No Images Yet'
+      expect(page).to have_link 'Upload Image'
+    end
```

Comment:
Split your tests into separate expectations. It may seem like a real pain, but it actually makes for more accurate documentation through your tests, and makes it easier to see exactly why something fails